spec/spec.md

```diff
@@ -641,10 +660,10 @@ module MyModule :
 ```
 
 For details on the syntax and semantics of the sub-field expression, sub-index
-expression, and statement groups, see sections [6.6](#subfields),
-[6.7](#subindices), and [5.3](#statement_groups).
+expression, and statement groups, see [@sec:sub-fields; @sec:sub-indices;
+@sec:statement-groups].
 
-### The Partial Connection Algorithm {#partial_connection_algorithm}
+### The Partial Connection Algorithm {#partial-connection-algorithm}
```

Comment:
why is this one still here `#partial-connection-algorithm`?

Reply:
Changed this back in a later commit.